api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -49,13 +54,23 @@ public void willStart() {
      */
     public void started() {
 		LocationUtils.createPersonAttributeTypeForLocation();
+		generateOpenMRSPrivilegeForLocations();
 	    createGlobalPropertyForLocationAttribute(LocationBasedAccessConstants.LOCATION_ATTRIBUTE_GLOBAL_PROPERTY_NAME,
 			    LocationBasedAccessConstants.PERSONATTRIBUTETYPE_UUID);
 	    createGlobalPropertyForLocationAttribute(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME,
 			    LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
 	    log.info("Location Based Access Control Module started");
     }
 
+	private void generateOpenMRSPrivilegeForLocations() {
+		UserService userService = Context.getUserService();
+		List<Location> openMrsLocations = Context.getLocationService().getAllLocations();
```

Comment:
Simply `allActiveLocations`?

Reply:
Currently, in user and patient Fragment controller, we are getting all the Locations, so here we are using all the Locations.
else we can change it to `getAllLocations(false)` to get only the non-retired one.